pkg/sdkserver/sdkserver_test.go

```diff
@@ -797,11 +790,8 @@ func TestSDKServerUpdateEventHandler(t *testing.T) {
 	t.Parallel()
 
 	// Acquire lock in order to be sure that
-	// no other parallel test turn on FeatureSDKWatchSendOnExecute
 	agruntime.FeatureTestMutex.Lock()
 	defer agruntime.FeatureTestMutex.Unlock()
-	err := agruntime.ParseFeatures(string(agruntime.FeatureSDKWatchSendOnExecute) + "=false")
```

Comment:
The tests `TestSDKServerUpdateEventHandler` and `TestSDKServerSendGameServerUpdate` were both running before with this feature gate _disabled_. This runs them with it on. Did you test that they work without modification?